site/zenodo_rdm/migrator/load.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022-2023 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Zenodo migrator files loader."""
+
+import contextlib
+from pathlib import Path
+
+import psycopg
+from invenio_rdm_migrator.load.postgresql import _ts
+from invenio_rdm_migrator.streams.files.models import (
+    FilesBucket,
+    FilesInstance,
+    FilesObjectVersion,
+)
+
+
+class ZenodoFilesLoad:
```

Comment:
Can extend invenio-rdm-migrator `Load` class (similar constructor). Then overrite `_load` and `run` to skip the prepare step.